shared/components/map.js

```diff
@@ -59,11 +70,36 @@ export default class Map extends Component {
     Actions.cameraView();
   }
 
+  updatePrevValues() {
+    this.prevLat = this.props.currentLocation.latitude;
+    this.prevLng = this.props.currentLocation.longitude;
+    this.prevZoom = Math.round(this.props.zoomLevel * 100) / 100;

```

Comment:
It is cool but just for string
toFixed does not only rounding but also toString()
The return value of it is **string**